packages/synapse-react-client/src/style/components/_submission-page.scss

```diff
@@ -29,7 +29,11 @@
     .SubmissionSummaryGrid {
       display: grid;
       column-gap: 10px;
-      grid-template-columns: auto auto;
```

Comment:
I have no idea what caused this change. Maybe recent changes to UserCard styles caused its width to exceed the parent here? Needs further investigation. In any case, the grid columns should be bounded with [minmax](https://developer.mozilla.org/en-US/docs/Web/CSS/minmax).

See also: https://css-tricks.com/preventing-a-grid-blowout/